src/constants/addresses.ts

```diff
@@ -53,6 +53,14 @@ const ARBITRUM_GOERLI_QUOTER_ADDRESSES = '0x1dd92b83591781D0C6d98d07391eea4b9a60
 const ARBITRUM_GOERLI_NONFUNGIBLE_POSITION_MANAGER_ADDRESSES = '0x622e4726a167799826d1E1D150b076A7725f5D81'
 const ARBITRUM_GOERLI_TICK_LENS_ADDRESSES = '0xb52429333da969a0C79a60930a4Bf0020E5D1DE8'
 
+// sepolia v3 addresses
+const SEPOLIA_V3_CORE_FACTORY_ADDRESSES = '0x0227628f3F023bb0B980b67D528571c95c6DaC1c'
```

Comment:
not relevant to PR - but kinda surprised these don't live in the SDK

Reply:
This does feel like a process that could be part of the protocol deployment scripts to save a lot of manual entry.